### ISSUE TITLE
Milestone 13 feedback

### ISSUE THREAD

**Issue by kandarpksk** [niccalle/cogs121#10]

**Comment by paulkim94** [comment]
Team K2: Guo-po

Paul Kim
- The most important thing to finish by next week is to add more information to each route in the home page such as the duration of the route and rating. Another important task would be to create some sort of landing page that informs the user about the application and its purpose. Some other tasks would be fixing bugs and improving the UI overall.

Judd Gatbonton
- The most important thing to finish by next week is the UI such as the interface between the map, video playback and the directions. Make sure everything is responsive as well. Clean up minor functionality to make sure it runs smoothly. Overall, it's looking good!

Ranvijay Singh
- The most important thing to finish by next week is any incomplete/buggy features that you guys are still working on(e.g. rating, playtime slider etc). Also, It would be nice to have a home/landing page that gives you some introduction to what the app is about. Overall, good work and great idea.

Anthony Grundle
- The mort important thing to finish by next week is to make sure the ratings show up on the list of routes, also some aesthetic fixes such as giving some padding around the video controls

**Comment by EmpressFavy** [comment]
Team K4

Andrew Fancett
I would like to see the directions as a drop down menu or some other button that doesn't always show, it can get pretty lengthy at times and The primary function of the app is to just visualize the route, not see the directions.

Christopher (Tyler) Sorensen
Feedback: I would reorganize the elements on  the page. They look like a collection of divs with little flow or logical spacing. I might also consider an estimated travel time if possible, as well as more granular directions (e.g. right on Prosser Ave. in 500 feet) if possible.

Drew Bartleman
I like the simple design, very clear and easy to see. I'm not entirely sure how well the rating function works - if a route is 99% similar to another route but takes a right turn at the end, is it a different route for rating purposes? The most important thing to add by next week is maybe a way to more easily change the way you get to your destination, during or before starting the journey there. Perhaps more functionality for biking? Bike trails? Probably impossible. Everything else looks great, good work!

Cynthia Hong
I think you can improve the UI by using a different font, you can grab a lot of cleaner ones on Google Fonts. Not too sure why I would wanna search for a route when making a route is pretty easy already. Also, is the route based on fastest time, or shorter distance?
